packages/neuron-wallet/src/controllers/app/index.ts

```diff
@@ -95,7 +95,13 @@ export default class AppController {
       minHeight: 600,
       show: false,
       backgroundColor: '#e9ecef',
-      icon: path.join(__dirname, '../../../assets/icons/icon.png'),
+      icon: nativeImage.createFromPath(
+        app.isPackaged
+          ? path.join(__dirname, '../../neuron-ui/icon.png')
```

Comment:
Why not use `assets/icons/icon.png` in packaged mode?

Reply:
`neuron-ui/icon.png` is the path [previously used](https://github.com/nervosnetwork/neuron/pull/1800/commits/539f7fe4f140a6b3dd41fe0928687ac43f5860eb), since `neuron-ui` is copied to dist folder during packaging, and `assets/icons/icon.png` was not packaged into asar.

Reply:
I see, it's kinda smelling but ok for now.